### ISSUE TITLE
Separate Columns in View Products page

### ISSUE THREAD

**Issue by benjanders** [kPOWz/Deviner#62]
Hey Guys-
 It looks like when we updated each product to have a Product ID and a Product Description, something happened and all of the attributes of the products got really scrambled. I believe these were all more or less correct before the most recent round of updates. It's fine, I have just received a few comments about the issue.

 I printed off a list of all 145 products we currently have in the database and my plan is to halt everyone on the screen printing side, give each person about 20 products to fix, and just knock it out in a quick session.

 So here's the question. In the View Products page, there are two columns. The Item column has the Product ID + Supplier, the other is a button to delete the product. How difficult would it be to break that Item column into three columns. Product ID, Product Description, and Vendor?

 It doesn't need to be sortable, doesn't need to be cleaned up, I personally just want separated columns so that we can tell which items have been updated. I don't know if it is a 10 minute fix or if that is a few-hour fix. If it is really quick, would that be possible to hammer out over the weekend? I'd love to round everyone up early next week to update the products.

**Comment by kPOWz** [comment]
Hm. 
At first glance this sounds like a miscommunication be myself and Ben or between Ben & John/rest of team. 

The intent that Ben and I went over was to automatically copy over the description on as many of the products as possible leaving the true item id as the itemid, but we could only do this for a certain portion of the products who's old Item IDs followed a pattern. 

The effect fell into the following three categories : 
1) no effect - this just means there was no item description to parse (if the old itemid had no '-')
2) best effect - perfecting parsable item id and item description (the old itemid had a '-' separating the id and desc)
3) wacky effect - approx 3 items affected  (the itemid had no dash between the itemid and desc but had a dash in the item desc   ex (made up): T201 Our favorite V-neck burnout tee ) 

I can verify that the intended effect was indeed what resulted, because I personally went over every single product in the database before and after the change to make sure I didn't see anything unintended or unexpected.

The only thing I found that I thought was strange was one product (fairly recently added /new) that was $1.79 and didn't have an item id ... )

That said, I would like a better understanding of what the perceived issue is (specifics), but we certainly can (& would want to ) add the description column to the products page anyway ...
